stats/opentelemetry/server_metrics.go

```diff
@@ -201,6 +212,16 @@ func (h *serverStatsHandler) TagRPC(ctx context.Context, info *stats.RPCTagInfo)
 		startTime: time.Now(),
 		method:    removeLeadingSlash(method),
 	}
+	if !isTracingDisabled(h.options.TraceOptions) {
+		var newAI *attemptInfo
+		ctx, newAI = h.traceTagRPC(ctx, info)
+		// Update the ai with values from updated attempt info.
+		newAI.startTime = ai.startTime
```

Comment:
similar comment. better to just update the fields of ai instead of populating a new variable and then copying